src/qibocal/auto/transpile.py

```diff
@@ -97,19 +98,65 @@ def execute_transpiled_circuit(
         backend,
         transpiler,
     )[0]
-
     return transpiled_circ, backend.execute_circuit(
         transpiled_circ, initial_state=initial_state, nshots=nshots
     )
 
 
+def get_natives(platform):
+    """
+    Return the list of native gates defined in the `platform`.
+    This function assumes the native gates to be the same for each
+    qubit and pair.
+    """
+    pairs = list(platform.pairs.values())[0]
+    qubit = list(platform.qubits.values())[0]
```

Comment:
It is often annoying that Python has no function to get the first element of an iterable (it is not LISP...), but in general, you should prefer using:
```suggestion
    pairs = next(iter(platform.pairs.values()))
    qubit = next(iter(platform.qubits.values()))
```
(if you have a list of 1M elements, you want to take the first one without creating a full duplicate...)